src/main/webapp/app/exercises/modeling/participate/modeling-submission.component.ts

```diff
@@ -401,6 +400,24 @@ export class ModelingSubmissionComponent implements OnInit, OnDestroy, Component
         }
     }
 
+    /**
+     * Find "Unreferenced Feedback" item for Result, if it exists.
+     */
+    get unreferencedFeedback(): Feedback[] | undefined {
+        if (this.assessmentResult && this.assessmentResult.feedbacks && Array.isArray(this.assessmentResult.feedbacks)) {
+            return this.assessmentResult.feedbacks.filter((feedbackElement) => feedbackElement.reference == undefined && feedbackElement.type === FeedbackType.MANUAL_UNREFERENCED);
+        }
+    }
+
+    /**
+     * Find "Referenced Feedback" item for Result, if it exists.
+     */
+    get referencedFeedback(): Feedback[] | undefined {
+        if (this.assessmentResult && this.assessmentResult.feedbacks && Array.isArray(this.assessmentResult.feedbacks)) {
```

Comment:
same as for unreferenced feedback `Array.isArray`

Reply:
👍